glib-macros/src/clone.rs

```diff
@@ -539,22 +539,66 @@ pub(crate) fn clone_inner(item: TokenStream) -> TokenStream {
     let return_kind = parse_return_kind(&mut parts);
     check_before_closure(&mut parts);
     let closure = get_closure(&mut parts);
-    let body = tokens_to_string(parts);
-
-    let x = format!(
-        "{{\n{}\nmove |{}| {{\n{}\nlet ____ret = {{ {} }};\n____ret\n}}\n}}",
```

Comment:
How did we ever think that was a good idea?

Reply:
Well, it makes the reading easier?